### PR TITLE
CI: switch to DMD 1.081.1

### PR DESCRIPTION
**Pull request by mihails-strasuns-sociomantic**

### CODE REVIEW

**Comment by nemanja-boric-sociomantic** [comment]
Too fast?

**Comment by mihails-strasuns-sociomantic** [comment]
Yep, needs regeneration of cachalot docker image.

File: .travis.yml [hunk 1]
```diff
@@ -19,10 +19,8 @@ env:
         # Make sure beaver is in the PATH
         - PATH="$(git config -f .gitmodules submodule.beaver.path)/bin:$PATH"
     matrix:
-        - DMD=1.080.0 DIST=xenial F=production AFTER_SCRIPT=1
-        - DMD=1.080.0 DIST=xenial F=devel
-        - DMD=1.081.0 DIST=xenial F=production
-        - DMD=1.081.0 DIST=xenial F=devel
+        - DMD=1.080.1 DIST=xenial F=production AFTER_SCRIPT=1
+        - DMD=1.080.1 DIST=xenial F=devel
```
Comment (leandro-lucarella-sociomantic) [inline .travis.yml]: Errrm... I guess you meant `1.081.1`, no `1.080.1`?

**Review by leandro-lucarella-sociomantic — COMMENTED** [review]
Maybe this is why it's not found?

**Comment by mihails-strasuns-sociomantic** [comment]
Fixed the version :)

**Review by nemanja-boric-sociomantic — APPROVED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -19,10 +19,8 @@ env:
         # Make sure beaver is in the PATH
         - PATH="$(git config -f .gitmodules submodule.beaver.path)/bin:$PATH"
     matrix:
-        - DMD=1.080.0 DIST=xenial F=production AFTER_SCRIPT=1
```
Comment (nemanja-boric-sociomantic) [inline .travis.yml:22]: Just to check, was removing v1.080.0 on purpose?

Comment (mihails-strasuns-sociomantic) [reply .travis.yml:22]: Yup, it was already done in v3.x.x branch

**Pull request merged**